src/CSSBattle/10_CloackedSpirits/WeiPin.html

```diff
@@ -0,0 +1,54 @@
+<div class='rect1'></div>
+<div class='cir1'></div>
+<div class='rect2'></div>
+<div class='cir2'></div>
+<div class='cir3'></div>
+<style>
+  body {
+    background: #62306D;
+    margin:0;
+    display: flex;
+    align-items: center;
+  }
+  .rect1 {
+    position: absolute;
+    width: 300;
```

Comment:
i think for value thats non-zero putting units like 'px' or '%' would be more dumb-proof for future...IMO